atom/browser/native_window_mac.mm

```diff
@@ -1672,8 +1672,12 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
 }
 
 void NativeWindowMac::AddTabbedWindow(NativeWindow* window) {
-  if ([window_ respondsToSelector:@selector(addTabbedWindow:ordered:)]) {
-    [window_ addTabbedWindow:window->GetNativeWindow() ordered:NSWindowAbove];
+  if (window_.get() == window->GetNativeWindow()) {
+    NSLog(@"Error: AddTabbedWindow cannot be called by a window on itself.");
```

Comment:
This doesn't go over JS logs, can we do `ThrowError` instead (pass in `mate::arguments` or something)